manage.py

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python
 import os
+# Do this before *anything*.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
```

Comment:
This was the one-line key change that fixes the issue. The rest is pep-8 stuff and redoing celeryd so it's easier to copy/paste fixes going forward.